### PR TITLE
Add minutes for 2021-02-24

### PR DESCRIPTION
**Pull request by sedders123**

### CODE REVIEW

**Review by RealOrangeOne — APPROVED** [review]

File: SR2021/2021-02-14.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Competition Team Meeting 2021-02-14
+
+## Present
+
+- Andy Barrett-Sprot
+- Peter Law
+- Jake Howard
+- James Seden Smith
+- Jenny Fletcher
+- Will Barber
```
Comment (PeterJCLaw) [inline SR2021/2021-02-14.md:10]: I tend to sort these alphabetically... looks like I didn't get that quite right this week.
```suggestion
- Andy Barrett-Sprot
- Jake Howard
- James Seden Smith
- Jenny Fletcher
- Peter Law
- Will Barber
```

File: SR2021/2021-02-14.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  - Should we run some more sooner?
+    - The weekend we’d want to do it is this weekend (so there’s two clear weeks between them), feels a bit too short notice.
+
+### Metrics
+  - Should we collect them now? Does it feel weird to collect them in the middle?
+  - There’s broadly two classes -- demographics and competition feedback
+    - Let’s capture these separately
+    - Let’s do a demographics survey now
+    - Jenny put together the survey (base on copy/pasta/edit from last year’s)
+    - Demographic metrics requirements from the trustees:<https://docs.google.com/document/d/1QfOpNkHKnWQiPdlgRsh1aJUtkCLGgrk6GfPfO8MMU5c/edit>
```
Comment (PeterJCLaw) [inline SR2021/2021-02-14.md:32]:
```suggestion
    - Demographic metrics requirements from the trustees: <https://docs.google.com/document/d/1QfOpNkHKnWQiPdlgRsh1aJUtkCLGgrk6GfPfO8MMU5c/edit>
```

File: SR2021/2021-02-14.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  - Let’s aim to finesse the ideas we have in mind already, rather than adding new things at this point
+
+  - Let’s get to a conclusion early in the Doing this weekend
+
+
+## Action Points
+
+- Jake: request access to info@ from the trustees (also poke them about Will’s SR account)
+- Jenny: prepare a demographics survey
+- Andy BS: email volunteers about upcoming Doings
```
Comment (PeterJCLaw) [inline SR2021/2021-02-14.md:58]: Would be great to have these turned into/linked to issues so we don't forget them :)

**Review by PeterJCLaw — COMMENTED** [review]
Nice, thanks for transferring these.
Logically approved (they match what was in the doc), but not actually approved because I wrote them (and because I'd like to see issues for the action points).

**Review by WillB97 — APPROVED** [review]

**Review by PeterJCLaw — APPROVED** [review]

**Review by WillB97 — APPROVED** [review]

**Pull request merged**